webclient/index.html

```diff
@@ -124,7 +124,29 @@
             if (scriptSha !== htmlSha) {
                 alert('Whoops. The cached files on your machine are out of sync with each other. That\'s our bad. Please hard-refresh the page?');
             }
+            let table = document.querySelector("#table");
+            if(chrome && chrome.storage) {
+                chrome.storage.onChanged.addListener((changes, namespace) => {
+                    if(namespace !== "local") {
+                        return;
+                    }
+                    parseLocalStorage();
+                });
+            }
+            function parseLocalStorage() {
+                chrome.storage.local.get('order', keyValueMap => {
+                    let order = Order.fromJSON(JSON.parse(keyValueMap.order));
+                    table.order = order;
+                });
+            }
+            let bitch = Number(window.localStorage.getItem("parseOrderBitch"));
+            function requestDom() {
+                bitch = (bitch === bitch ? bitch: 0) +1;
+                chrome.storage.local.set({parseOrderBitch: bitch});
+            }
+            document.getElementById("parse").addEventListener("click", requestDom);
         </script>
+        <button id="parse">Parse Dom? Wut?</button>
```

Comment:
shouldn't be in index, but I couldn't get it to reliably notify the content script with just calling the function once